x-pack/plugins/cases/public/common/utils.test.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { parseUrlQueryParams } from './utils';
+import { stringify } from 'query-string';
+import { DEFAULT_QUERY_PARAMS } from '../containers/use_get_cases';
+
+describe('utils', () => {
```

Comment:
Let's add some tests for:

- parsedUrlParams.page and parsedUrlParams.pageSize equaling empty strings
- parsedUrlParams.page and parsedUrlParams.pageSize equaling 0